datasets/hippocorpus/hippocorpus.py

```diff
@@ -0,0 +1,153 @@
+}
+"""
+
+_DESCRIPTION = """\
+To examine the cognitive processes of remembering and imagining and their traces in language, we introduce Hippocorpus, a dataset of 6,854 English diary-like short stories about recalled and imagined events. Using a crowdsourcing framework, we first collect recalled stories and summaries from workers, then provide these summaries to other workers who write imagined stories. Finally, months later, we collect a retold version of the recalled stories from a subset of recalled authors. Our dataset comes paired with author demographics (age, gender, race), their openness to experience, as well as some variables regarding the author's relationship to the event (e.g., how personal the event is, how often they tell its story, etc.).
+"""
+
+_HOMEPAGE = "https://msropendata.com/datasets/0a83fb6f-a759-4a17-aaa2-fbac84577318"
+
+
+class Hippocorpus(datasets.GeneratorBasedBuilder):
+
+    VERSION = datasets.Version("1.1.0")
+
+    @property
+    def manual_download_instructions(self):
+        return """\
+  To use hippocorpus you need to download it manually. Please go to its homepage (https://msropendata.com/datasets/0a83fb6f-a759-4a17-aaa2-fbac84577318)and login. Extract all files in one folder and use the path folder in datasets.load_dataset('hippocorpus', data_dir='path/to/folder/folder_name')
+  """
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # datasets.features.FeatureConnectors
+            features=datasets.Features(
+                {
+                    # These are the features of your dataset like images, labels ...
+                    "AssignmentId": datasets.Value("string"),
+                    "WorkTimeInSeconds": datasets.Value("string"),
+                    "WorkerId": datasets.Value("string"),
+                    "annotatorAge": datasets.Value("string"),
+                    "annotatorGender": datasets.Value("string"),
+                    "annotatorRace": datasets.Value("string"),
+                    "distracted": datasets.Value("string"),
+                    "draining": datasets.Value("string"),
```

Comment:
Could this be integers instead of strings ?

Reply:
Hi @lhoestq, I have changed all the (5-point Likert) columns from string to float. A few checks have failed due to` FAILED tests/test_file_utils.py::TempSeedTest::test_tensorflow` .